OSDOCS#7519: Custom RHCOS image for control plane and compute machines (#63767)
openshift/openshift-docs: modules/installation-gcp-marketplace.adoc

:_content-type: PROCEDURE
[id="installation-gcp-marketplace_{context}"]
= Using the GCP Marketplace offering

Using the GCP Marketplace offering lets you deploy an {product-title} cluster, which is billed on pay-per-use basis (hourly, per core) through GCP, while still being supported directly by Red{nbsp}Hat.

By default, the installation program downloads and installs the {op-system-first} image that is used to deploy compute machines. To deploy an {product-title} cluster using an {op-system} image from the GCP Marketplace, override the default behavior by modifying the `install-config.yaml` file to reference the location of GCP Marketplace offer.

.Prerequisites

* You have an existing `install-config.yaml` file.

.Procedure

. Edit the `compute.platform.gcp.osImage` parameters to specify the location of the GCP Marketplace image:

> ** Set the `project` parameter to `redhat-marketplace-public`

Reviewer: ```suggestion
** Set the `project` parameter to `redhat-marketplace-public`.
```

> ** Set the `name` parameter to one of the following offers:

Reviewer: Should "offers" be "offerings"?

Or maybe:

```suggestion
** Set the `name` parameter to one of the following values, depending on your offering:
```

+

> {product-title}:: `redhat-coreos-ocp-413-x86-64-202305021736`

Reviewer: I might put the value first and then the offering. I'm sure people will understand, but at first glance it almost looks like you put "OpenShift Container Platform" as the value.

Author: This formatting has been in the docs for a while now. I would prefer to leave it as is.

> {opp}:: `redhat-coreos-opp-413-x86-64-202305021736``

Reviewer: ```suggestion
{opp}:: `redhat-coreos-opp-413-x86-64-202305021736`
```
